### PR TITLE
files added for lapindrome_checker

### PR DESCRIPTION
**Pull request by tbhaxor**
#1677 
files named `lapindrome_checker.cpp` , `lapindrome_checker.cs` and `lapindrome_checker.py` are added and ready to be merged 👍

### CODE REVIEW

File: code/string-algorithms/lapindrome_checker/lapindrome_checker.cpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  string left = str.substr(0,mid);
+  string right = str.substr(delim, len);
+
+  // sorting two strings
+  sort(left.begin(), left.end());
+  sort(right.begin(), right.end());
+
+  // equating and returning
+  if(left==right)
+    return true;
```
Comment (arnavb) [inline code/string-algorithms/lapindrome_checker/lapindrome_checker.cpp]: Could be simplified to `return left == right`

File: code/string-algorithms/lapindrome_checker/lapindrome_checker.cs [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            // slicing two strings
+            string left = str.Substring(0, mid);
+            string right = str.Substring(delim);
+
+            // sorting strings using lambda expression
+            left = String.Concat(left.OrderBy(x => x));
+            right = String.Concat(right.OrderBy(x => x));
+
+            // testing if equal
+            if (left == right)
```
Comment (arnavb) [inline code/string-algorithms/lapindrome_checker/lapindrome_checker.cs]: Here also, could simplify to `return left==right`

File: code/string-algorithms/lapindrome_checker/lapindrome_checker.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    # sorting  string
+    left_sorted = sorted(left) # this will return a list
+    right_sorted = sorted(right) # this will return a list
+
+    # converting list to string
+    left = "".join(left_sorted)
+    right = "".join(right_sorted)
+
+    # equating
+    if left == right:
```
Comment (arnavb) [inline code/string-algorithms/lapindrome_checker/lapindrome_checker.py]: Here as well.

**Comment by tbhaxor** [comment]
@arnavb and what about `return false`
btw i am simplifying 😄

**Comment by tbhaxor** [comment]
@arnavb bro i have simplified `return` statements and are ready to be merged
thnx for notifying me

**Pull request closed without merging**